tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -4857,19 +4857,10 @@ Feature: an user gets the resources shared to them
     """
     {
       "type": "object",
-      "required": ["hidden", "mount_point"],
+      "required": ["@UI.Hidden"],
       "properties": {
-        "hidden": {
+        "@UI.Hidden": {
```

Comment:
I wonder why the CI was green previously

Reply:
The PR (https://github.com/owncloud/ocis/pull/8939) was not rebased and CI was not run against the latest tests added.